force-app/main/default/lwc/ldsCreateRecordWithRecordInput/ldsCreateRecordWithRecordInput.html

```diff
@@ -0,0 +1,51 @@
+<template>
+    <lightning-card
+        title="LdsCreateRecordWithRecordInput"
+        icon-name="standard:record"
+    >
+        <div class="slds-var-m-around_medium">
+            <template if:false={error}>
+                <template if:true={nameEditable}>
```

Comment:
some of this markup could be reduced if you had a single field conditionally visible.

Reply:
Change applied.